### DISCUSSION TITLE
Error constructing CesiumWidget (webGL)

### DISCUSSION THREAD

**Discussion by PetukhovArt** [zouyaoji/vue-cesium#586] [Q&A]
vue-cesium 3.2.8 
vue 3.4.30 
cesium 1.119
Windows NT 10.0; Win64; x64 / Chrome/126.0.6478.114 / Electron/31.1.0

how i import cesium (core), now we have files of cesium build locally : 

```
app.use(VueCesium, {
  locale: enUS,
  cesiumPath: `path to cesium public folder /Cesium.js`,
});
```

after we **switch our electron app with cesium widget to vite from webpack** , sometimes we have production and dev error and app crash 

![image_2024-07-17_10-33-02](https://github.com/user-attachments/assets/3d3a1b23-b11e-4149-8a70-4de04548d3e5)

have you any suggestions , how i can fix this ?

**Comment by zouyaoji** [comment]
I suggest you try updating your graphics card driver. Can the online document viewer open normally?

https://zouyaoji.top/vue-cesium/#/zh-CN/component/vc-viewer

**Reply by PetukhovArt** [reply]
i tried for drivers, yes, online work normally .
an everything was fine on webpack 
i think , may be some trouble with import Cesium as local asset ( Cesium build in my public folder) and Viewer can't get webGl context 

if i use standard  import , widget is loading normally  : 

`
app.use(VueCesium, {
  locale: enUS,
});
`

**Reply by zouyaoji** [reply]
Could you provide a project that can reproduce the issue? I will take a look at it when I have time.

**Reply by PetukhovArt** [reply]
thinking... how to do that, i have i large electron app with this problem
